lib/include/tick/array/serializer.h

```diff
@@ -0,0 +1,20 @@
+
+
+#include "sparsearray.h"
+#include "sparsearray2d.h"
+
+class Serializer{
```

Comment:
Do we really still need this class? As it has no state we should be able to expose simply array_to_file and sparse2d_to_file methods.

Reply:
I'm not a fan of global public functions but I guess for swig it makes sense.